posthog/temporal/batch_exports/batch_exports.py

```diff
@@ -194,7 +201,11 @@ def iter_records(
         include_events_statement = ""
         events_to_include_tuple = ()
 
-    timestamp_predicates = timestamp_predicates = get_timestamp_predicates_for_team(team_id)
+    if is_backfill:
```

Comment:
NIT: could make this a separate func as it's repeated

Reply:
Will follow-up with a PR to clean this up.